tox.ini

```diff
@@ -49,28 +50,28 @@ pip_pre = True
 commands = python -b -W always -m coverage run -m django test -v2 {posargs:tests}
 
 
-[testenv:py{38,39,310,311,312}-dj{42,50,51,main}-{postgresql,psycopg3}]
+[testenv:py{38,39,310,311,312,313}-dj{42,50,51,main}-{postgresql,psycopg3}]
```

Comment:
You can remove 38 here and below as well.

Reply:
So still struggling a bit to get the matrix running correctly. I think we only want py313 with dj51 and main, and then postgres and postgis DB backends but using psycopg as the adapter library.